### DISCUSSION TITLE
2-m relation

### DISCUSSION THREAD

**Discussion by kdev** [prisma/prisma#5805] [Q&A] [answered]
Hey 👋
I have a model for sanctions that can be invoced on someone, it currently looks like this:
```prisma
model Sanction {
    id              Int       @id @default(autoincrement())
    memberId        String
    moderatorId     String
    reason          String?   @default("N/A")
    type            String
    timestamp       DateTime? @default(now())
    sanctioning_end DateTime?
    terminates      Int?
}
```
And I have a model for a user
```prisma
model User {
    id         String     @id
    language   String?    @default("en")
}
```
I know need to connect a user to the `memberId ` as well as the `moderatorId`.
I tried this but I'm not really sure if this is how it should be done (?):
```prisma
model Sanction {
    id              Int       @id @default(autoincrement())
    memberId        String
    moderatorId     String
    reason          String?   @default("N/A")
    type            String
    timestamp       DateTime? @default(now())
    sanctioning_end DateTime?
    terminates      Int?

    User      User @relation("user", fields: [memberId], references: [id])
    Moderator User @relation("moderator", fields: [moderatorId], references: [id])
}

model User {
    id         String     @id
    language   String?    @default("en")
    sanctions  Sanction[] @relation("user")
    sanctioned Sanction[] @relation("moderator")
}
```

Thanks in advance!!

**Comment by ryands17** [comment, marked as answer]
@KokosnussDEV 👋 

Yes this is the correct way of specifying multiple relations to the same model as this is the way described in the [docs](https://www.prisma.io/docs/concepts/components/prisma-schema/relations#disambiguating-relations) as well :)

**Reply by kdev** [reply]
Okay, perfect, then I got it. Thanks for the fast reply!!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
